### PR TITLE
Add a docstring to export_data_command

### LINKED ISSUE #14530: Document how to export user data
**Description:**

There is a command to export user data: `python -m synapse.app.admin_cmd -c <config_file> export-data <user_id>` but as far as I can tell it is not documented anywhere.

### PR DESCRIPTION
**Pull request by krish-bista**
Fixes #14530.

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [x] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
* [x] [Code style](https://matrix-org.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

**Comment by clokep** [comment]
@krish-bista Please [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off) on your PR.

File: synapse/app/admin_cmd.py [hunk 1]
```diff
@@ -101,8 +101,18 @@ class AdminCmdServer(HomeServer):
...
+        This code defines an asynchronous function named "export_data_command" that 
+        takes in two arguments: "hs" which is an instance of the "HomeServer" class, 
+        and "args" which is an instance of the "argparse.Namespace" class. The function
+        exports data for a specific user by using the "export_user_data" method of 
+        the "HomeServer" class's "get_admin_handler" attribute. The method takes in 
+        two arguments: the "user_id" which is passed in from the "args" variable, 
+        and "FileExfiltrationWriter(user_id, directory=directory)" which is a class that writes 
+        the exported data to a specified directory. The result of the "export_user_data" method is 
+        stored in the "res" variable, and then printed out.
+    """
```
Comment (DMRobertson) [inline synapse/app/admin_cmd.py:115]: There is a rough convention for docstrings that we adhere to. A nice example is https://github.com/matrix-org/synapse/blob/6a6e1e8c0711939338f25d8d41d1e4d33d984949/synapse/api/ratelimiting.py#L95-L133

We would accept this change if your comment was in a format like that.

**Review by DMRobertson — COMMENTED** [review]
Note that the issue mentioned in the branch name has been closed by #14883.

We always welcome docstring improvements, however: docstrings are not user-facing documentation, so this wouldn't suffice to close #14530.

**Comment by clokep** [comment]
Closing per missing sign-off.

**Pull request closed without merging**